easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-foss-2016b.eb

```diff
@@ -0,0 +1,44 @@
+# authors: Kenneth Hoste (Ghent University), George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# foss-2016b modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'EMBOSS'
+version = '6.6.0'
+
+homepage = 'http://emboss.sourceforge.net/'
+description = """EMBOSS is 'The European Molecular Biology Open Software Suite'.
+ EMBOSS is a free Open Source software analysis package specially developed for
+ the needs of the molecular biology (e.g. EMBnet) user community."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['ftp://emboss.open-bio.org/pub/EMBOSS/']
```

Comment:
@verdurin we should also include `ftp://emboss.open-bio.org/pub/EMBOSS/old/%(version_major_minor)s.0` as alternate download URL to avoid breaking the download when a newer version is released?